datadog_checks_base/datadog_checks/checks/prometheus/mixins.py

```diff
@@ -39,40 +41,76 @@ class PrometheusScraperMixin(object):
     # This class is not supposed to be used by itself, it provides scraping behavior but
     # need to be within a check in the end
 
-    UNWANTED_LABELS = ["le", "quantile"]  # are specifics keys for prometheus itself
+    UNWANTED_LABELS = ['le', 'quantile']  # are specifics keys for prometheus itself
     REQUESTS_CHUNK_SIZE = 1024 * 10  # use 10kb as chunk size when using the Stream feature in requests.get
 
     def __init__(self, *args, **kwargs):
+        # Initialize AgentCheck's base class
         super(PrometheusScraperMixin, self).__init__(*args, **kwargs)
 
-        # The scraper needs its own logger
-        self.log = logging.getLogger(__name__)
-
         # message.type is the index in this array
         # see: https://github.com/prometheus/client_model/blob/master/ruby/lib/prometheus/client/model/metrics.pb.rb
         self.METRIC_TYPES = ['counter', 'gauge', 'summary', 'untyped', 'histogram']
 
+    def create_mixin_configuration(self, instance=None):
```

Comment:
Can this be called `create_scaper_configuration` or `create_endpoint_configuration` so it's more clear that you should create one of these per-endpoint.